kubernetes-itests/src/test/java/io/fabric8/commons/Delete.java

```diff
@@ -0,0 +1,57 @@
+package io.fabric8.commons;
+
+import io.fabric8.kubernetes.client.KubernetesClient;
+import io.fabric8.openshift.client.OpenShiftClient;
+
+import java.util.concurrent.Callable;
+
+public class Delete<T> implements Callable<Boolean> {
+
+  private KubernetesClient client;
+  private String name;
+  private String namespace;
+  private Class<T> typeParameterClass;
+
+  public Delete(Class<T> typeParameterClass, KubernetesClient client, String name, String namespace) {
+    this.typeParameterClass = typeParameterClass;
+    this.client = client;
+    this.name = name;
+    this.namespace = namespace;
+  }
+
+  @Override
+  public Boolean call() throws Exception {
```

Comment:
Are you sure that Exception is thrown from this method? if not, please remove this.

Reply:
forgot to omit this